specifyweb/frontend/js_src/lib/components/FormEditor/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -45,7 +45,7 @@ exports[`Can edit a form definition 2`] = `
 "<?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?>
 <viewset name=\\"Fish Views\\" xmlns:xsi=\\"http://www.w3.org/2001/XMLSchema-instance\\">
 	<views>
-		<view name=\\"Accession\\" class=\\"edu.ku.brc.specify.datamodel.Accession\\" busrules=\\"edu.ku.brc.specify.datamodel.busrules.AccessionBusRules\\" resourcelabels=\\"false\\">
+		<view name=\\"Accession\\" class=\\"edu.ku.brc.specify.datamodel.Accession\\" busrules=\\"edu.ku.brc.specify.datamodel.busrules.\\" resourcelabels=\\"false\\">
```

Comment:
This is incorrect

Reply:
From `common.views.xml`:
```xml
        <view name="Accession"
              class="edu.ku.brc.specify.datamodel.Accession"
              busrules="edu.ku.brc.specify.datamodel.busrules.AccessionBusRules"
        >
            <desc><![CDATA[The Accession form.]]></desc>
            <altviews>
                <altview name="Accession View" viewdef="Accession" mode="view" default="true"/>
                <altview name="Accession Edit" viewdef="Accession" mode="edit"/>
            </altviews>
        </view>
```

Reply:
Fixed in https://github.com/specify/specify7/pull/4816/commits/46a98f081894f0cb2c4b66834bcdc0e652d3176d